packages/patternfly-4/react-catalog-view-extension/src/components/CatalogTile/CatalogTile.tsx

```diff
@@ -143,7 +143,7 @@ export class CatalogTile extends React.Component<CatalogTileProps> {
         </CardHeader>
         <CardBody className="catalog-tile-pf-body">
           <div className="catalog-tile-pf-description">
-            <span className={classNames({'truncated': isTruncated})}>
+            <span className={classNames('co-line-clamp', {'truncated': isTruncated})}>
```

Comment:
If we are auto-truncating via the `co-line-clamp` then we should not be truncating the string at all and no need for the `truncated` class. 

We should remove the truncation properties (and update the text in the example tiles)